src/main/java/org/jpeek/graph/XmlGraph.java

```diff
@@ -104,28 +105,61 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
 
     /**
      * Serialize method call to a string.
-     * @param call Call operation as XML
-     * @return A string representation of the call operation
-     * @throws IOException If fails
-     * @todo #413:30min The following 2 private methods (call and args) could be extracted
-     *  in two new class to encapsulate logic of serializing an XML method call
-     *  to its string representation; and the logic of serializing an XML method arguments
-     *  to its string representation.
+     *
+     * @since 1.0
+     * @todo #440:30min This class XMLCallText and the following one
+     *  XMLMethodArgsText should be made public and tests should be added
+     *  to validate both their behaviours.
      */
-    private static String call(final XML call) throws IOException {
-        return new Joined(
-            "", call.xpath("name/text()").get(0),
-            ".", XmlGraph.args(call)
-        ).asString();
+    private static final class XmlCallText implements Text {
+
+        /**
+         * XML Call operation.
+         */
+        private final XML call;
+
+        /**
+         * Ctor.
+         *
+         * @param call Call operation as XML.
+         */
+        XmlCallText(final XML call) {
+            this.call = call;
+        }
+
+        @Override
+        public String asString() throws IOException {
+            return new Joined(
+                "", this.call.xpath("name/text()").get(0),
+                ".", new XmlMethodArgsText(call).asString()
+            ).asString();
+        }
     }
 
     /**
      * Serialize method arguments to a string.
-     * @param method Method as XML
-     * @return A string representation of the method args
-     * @throws IOException If fails
+     *
+     * @since 1.0
      */
-    private static String args(final XML method) throws IOException {
-        return new Joined(":", method.xpath("args/arg/@type")).asString();
+    private static final class XmlMethodArgsText implements Text {
```

Comment:
@victornoel I believe `Text` suffix is redundant. `XmlMethodArgs` sound cleaner.